ts/screens/profile/NewProfileScreen.tsx

```diff
@@ -99,9 +110,13 @@ const NewProfileScreen = () => {
           subTitle={getValueFromNewProfilePot("email")}
           leftIcon={EmailIcon}
           hideIcon
-          isLastItem
           testID="email"
         />
+        <PreferencesListItem
+          title={I18n.t("profile.data.deletion.title")}
+          description={I18n.t("profile.data.deletion.description")}
+          rightElement={<RemoteSwitch value={isUserDataProcessingDeletePot} />}
```

Comment:
```suggestion
          rightElement={<RemoteSwitch value={isUserDataProcessingDeletePot} onValueChange={_ => constVoid} />}
```
just as a suggestion to if you want you can add a placeholder function which won't actually cause any side effects but will work only as a TODO to remove in a future implementation

Reply:
Thanks!
Next time I will use it whenever I need to add a placeholder function 🙌